File: targets/html5/modules/native/monkeytarget.js

function BBMonkeyGame( canvas ){
	BBHtml5Game.call( this,canvas );
}

BBMonkeyGame.prototype=extend_class( BBHtml5Game );

BBMonkeyGame.Main=function( canvas ){

	var game=new BBMonkeyGame( canvas );

	try{

		bbInit();
		bbMain();

	}catch( ex ){
	
		game.Die( ex );
		return;
	}

	if( !game.Delegate() ) return;
	
	game.Run();
}

File: targets/__deprecated__/psm/modules/native/monkeytarget.cs

public class BBMonkeyGame : BBPsmGame{

	public static void Main( String[] args ){
	
		BBMonkeyGame game=new BBMonkeyGame();
		
		try{
		
			bb_.bbInit();
			bb_.bbMain();
			
		}catch( Exception ex ){

			if( game.Die( ex ) ) throw;
			return;
		}
		
		if( game.Delegate()==null ) return;
		
		game.Run();
	}
}
